src/Map.php

```diff
@@ -41,7 +41,9 @@ public function asVersion($version = null) {
     public function set($code, $value) {
         $this->_map[$code] = $value;
     }
-    private function _unset($code) {
+
+    public function _unset($code)
+    {
```

Comment:
What's the reason for the interface change here? If we expect to use it externally we should drop the leading '_' from the name.